lib/api/controllers/cliController.js

```diff
@@ -21,7 +21,7 @@ function CliController (kuzzle) {
     };
 
     kuzzle.services.list.broker.listen(kuzzle.config.queues.cliQueue, this.onListenCB);
-    kuzzle.pluginsManager.trigger('log:info', 'CLI controller initialized');
+    return kuzzle.pluginsManager.trigger('log:info', 'CLI controller initialized');
```

Comment:
Returning the response of `log:info` event make it available to pipe plugins.

I think we should not return on this promise to let it run in background.
So I suggest to only `return null` here